src/App.js

```diff
@@ -128,7 +97,15 @@ const App = () => {
             7-day moving average of COVID-19 <br />deaths per million people
           </div>
         </h1>
-        <div id='chart'></div>
+        {
+          [geolocationStates.loaded, geolocationStates.disallowed].includes(geolocationState)
+          ? (
+            <div id='chart'></div>
```

Comment:
If this is rendered conditionally, let's make `drawChart()` bail out if the element it's trying to render to does not exist